### DISCUSSION TITLE
What's your most anticipated feature?

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#10250] [Polls]
There are endless bug fixes and enhancements, but we also need to move forward with the unfinished features.

**Comment by robidog** [comment]
I would also love to see a ready-to-deploy and self-updating virtual appliance for the server component.

**Reply by Affection4036** [reply]
As a new MSP this one is huge to me, next to cross platform copy and paste.

**Reply by xlionjuan** [reply]
https://containrrr.dev/watchtower/

**Comment by tirsojrp** [comment]
**Show other users cursors on host pc** can be a deal breaker for those who provide remote assistance and are restricted to view mode only.

**Reply by rustdesk** [reply]
![image](https://github.com/user-attachments/assets/295972e9-becb-4630-be34-54db5f31be51)

**Reply by tirsojrp** [reply]
Sorry,

**_Show other users cursors on host pc_**  This is the feature I wanted to point out.
